contracts/scripts/deploy.ts

```diff
@@ -20,7 +20,21 @@ async function main() {
   await summa.deployed();
 
   console.log(`Summa deployed to ${summa.address}`);
+  
+  const signers = await ethers.getSigners();
+  
+  for (let i = 1; i < 4; i++) {
+    console.log(signers[i].address);
+    await ethers.provider.send("hardhat_setBalance", [signers[i].address, "0x2d515"])
+  }
+
+  const mockERC20 = await ethers.deployContract("MockERC20");
+  await mockERC20.deployed();
+
+  console.log(`MockERC20 deployed to ${mockERC20.address}`);
+  await mockERC20.mint("0x90F79bf6EB2c4f870365E785982E1f101E93b906", 556863);
```

Comment:
Is it possible to keep all the mock balance assignment and minting within the tests? Something like a single `initialize_anvil` function would suffice. Usually, a separate deployment script is written as a general-purpose one, assuming it can be used for real deployments, so it wouldn't do the mock minting.

Reply:
Ah I see, we can remove this and the cli app also using `Anvil` instead of `hardhat`.
Thanks for point out

Reply:
It removed at 9ada335